backend/src/modules/player-profiles/core/domain/CretaePlayerProfile.ts

```diff
@@ -0,0 +1,26 @@
+import {DomainEvent} from "../../../../shared/domain/event/DomainEvent";
+import {PlayerProfileWasCreated} from "./event/PlayerProfileWasCreated";
+
+export function createPlayerProfile(
+    command: {
+        playerId: string;
+        firstName: string;
+        lastName: string;
+        emailAddress: string;
```

Comment:
Możemy mieć 2 profile graczy z tym samym mailem albo telefonem :)? To trzeba obgadać z Product Ownerem.

Reply:
dodałem //TODO w pliku PlayerProfile w domain w funkcji onPlayerProfileWasCreated :-)